.release/security-scan.hcl

```diff
@@ -13,6 +13,6 @@ binary {
   nvd        = false
 
   secrets {
-    all = true
+    all = false
```

Comment:
Oh, we can flip this back to `true`. The false positive problem has a fix that will be available soon as well 🤞